src/redist/targets/Crossgen.targets

```diff
@@ -5,14 +5,14 @@
     
     <PropertyGroup>
       <RuntimeNETCoreAppPackageName>runtime.$(SharedFrameworkRid).microsoft.netcore.app</RuntimeNETCoreAppPackageName>
-      <_crossDir Condition="'$(Architecture)' == 'arm64'">/x64_arm64</_crossDir>
+      <_crossDir Condition="'$(Architecture)' == 'arm64' and '$(BuildArchitecture)' != 'arm64'">/x64_arm64</_crossDir>
```

Comment:
Ideally, this would be `'$(BuildArchitecture)' == 'x64'` but that wouldn't work (as far as I understand) for VS. Any suggestions?